packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -18,11 +18,29 @@ import {FocusRing, useFocusRing} from '@react-aria/focus';
 import {GridNode} from '@react-types/grid';
```

Comment:
This is the one file we need to figure out how to undo our prettier changes, sorry! We'll try to figure that out soon. Unfortunately it feels like we'll have to do this manually unless someone has a better idea.

Reply:
without knowing how many actual changes you made, I don't really know, you could copy our existing code and just apply your changes in manually if there aren't too many
I assume it's trying to fit the 80char column scheme?

Reply:
That's probably a good idea, @snowystinger and yes you're probably right about the 80char column scheme.